frontend/src/scenes/experiments/Experiment.tsx

```diff
@@ -131,6 +133,24 @@ export function Experiment(): JSX.Element {
                 <div>Owner: {experimentData.created_by?.first_name}</div>
                 <div>Feature flag key: {experimentData?.feature_flag_key}</div>
             </div>
+
+            {experimentResults && (
+                <BindLogic
+                    logic={insightLogic}
+                    props={{
+                        dashboardItemId: undefined,
+                        filters: { ...experimentResults.filters, insight: 'FUNNELS', display: 'FunnelViz' },
```

Comment:
filters currently default to returning `ActionsLineGraph` and `TRENDS`. Assuming these are defaults, make sure FE always sends these two params on Experiment creation